### ISSUE TITLE
RSS widget is not working

### ISSUE THREAD

**Issue by vkorn** [home-assistant/appdaemon#72]
Greetings, 

I'm using v2 and trying to add RSS widget to my dashboard. 
Even if everything is configured using examples, RSS is not displaying. 
AppDaemon config 
```yml
AppDaemon:
  cert_verify: False
  logfile: STDOUT
  errorfile: STDERR
  threads: 10
  app_dir: /config/hadaemon/apps
  rss_feeds: 
    - feed: http://rss.cnn.com/rss/cnn_topstories.rss
      target: cnn
  rss_update: 600
```

And in dashboard I have 
```yml
rss_cnn: 
  widget_type: rss
  title: CNN
  entity: cnn
  interval: 5
```

I couldn't find anything related to RSS in logs.

**Comment by acockburn** [comment]
Does the widget show up at all? Do you see anything in the Javascript log?

**Comment by mezz64** [comment]
Did you restart appdaemon after enabling rss?  I had to do a restart before it started working for me.

**Comment by vkorn** [comment]
@acockburn Widget is showing up empty. And I forgot to check JS errors, sorry, here's what I have every page load:  
```
dashboard.js:137 Uncaught TypeError: Cannot use 'in' operator to search for 'icon' in undefined
    at Object.success (dashboard.js:137)
    at i (jquery-3.1.1.min.js:2)
    at Object.fireWith [as resolveWith] (jquery-3.1.1.min.js:2)
    at A (jquery-3.1.1.min.js:4)
    at XMLHttpRequest.<anonymous> (jquery-3.1.1.min.js:4)
```

And seems that every `interval` minutes 
```
Uncaught TypeError: Cannot set property 'cnn' of undefined
    at baserss.WidgetBase.on_ha_data (http://cp.local.io/javascript/dashboard.js:175:47)
    at http://cp.local.io/javascript/dashboard.js:17:30
    at Array.forEach (native)
    at a.webSocket.onmessage (http://cp.local.io/javascript/dashboard.js:13:30)
    at HTMLDivElement.<anonymous> (http://cp.local.io/javascript/reconnecting-websocket.min.js:1:836)
    at WebSocket.h.onmessage (http://cp.local.io/javascript/reconnecting-websocket.min.js:1:2312)
```

@mezz64 , yes

**Comment by acockburn** [comment]
I've put in a fix for one of the errors in the latest dev branch if you want to check it out and let me know if it resolves it.

If you prefer not to look at the dev branch, , it will be in the next release.

**Comment by acockburn** [comment]
Fix is now in 2.0.3 if you want to take a look.

**Comment by vkorn** [comment]
Sorry, didn't have chance to check earlier. Works like a charm now. Thank you!
